cmd/zinject/zinject.c

```diff
@@ -159,8 +159,6 @@
 libzfs_handle_t *g_zfs;
 int zfs_fd;
 
-#define	ECKSUM	EBADE
```

Comment:
as i can see, for linux we have :
`lib/libspl/include/os/linux/sys/errno.h`
where ECKSUM defined.
it is not the same place on dilos and i'd like to use:
```
#ifndef ECKSUM
#define	ECKSUM	EBADE
#endif
```
if it is impact freebsd.

Reply:
There’s no EBADE on FreeBSD. Please use the established layout.

Reply:
Part of the concern is that `EBADE` isn't defined as a valid errno on all platforms.  This is why the `ECKSUM` define was moved in to the platform code.  Could something similar be done on dilos?

Reply:
i understood. well, it's confused to me because it said in subject about Linux specific `ECKSUM `.
i don't know where is good place for `ECKSUM ` right now and for ZFS code specific and reduce collisions with others definitions - can we map/remap it it to ZFS related name - something like :
```
#define ZFS_ ECKSUM _VALID_VAR
```
and we can map it to valid platform specific `_VALID_VAR` where we need.
if i need update system/public header with definition, i'd like to avoid collisions

Reply:
Then for the moment, I'd suggest not pulling in this change.  Or adding it to one of the zfs headers, after the `zfs_errno_t` enum might be a good place.  For the moment, we want to avoid doing something like rename ECKSUM since we'd need to update the callers.  And, as far as I know, `ECKSUM` doesn't collide with any existing errnos.